.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
> I could in theory as part of the interface make it print out any tests that were listed to be skipped but not actually skipped before the test suite is run. This would allow us to know if there are stray tests in the array no longer needed. 

I like that idea!